utils/defi.ts

```
@@ -1,278 +1,304 @@
+                return `https://app.nostra.finance/lend-borrow/${first}/deposit`
+            }
+            break
+
+        case "vesu":
+            if (first) {
+                return baseUrl
+            }
+            break
+
+        case "haiko":
+            console.log(parseTokenPair(poolInfo), "login this", first && second)
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)